app/javascript/react/components/App.js

```diff
@@ -1,18 +1,19 @@
 import React from 'react'
 import {BrowserRouter, Route, Switch} from 'react-router-dom'
+
 import GamesIndexContainer from './GamesIndexContainer'
 import NewGameContainer from './NewGameContainer'
 import GameShowContainer from './GameShowContainer'
-
-
+import UserShowContainer from './UserShowContainer'
 export const App = (props) => {
   return (
     <BrowserRouter>
       <Switch>
         <Route exact path='/games' component={GamesIndexContainer}/>
         <Route exact path='/games/new' component={NewGameContainer}/>
         <Route exact path='/' component={GamesIndexContainer}/>
-        <Route path='/games/:id' component={GameShowContainer}/>
+        <Route exact path='/games/:id' component={GameShowContainer}/>
+        <Route exact path='/users/:id' component={UserShowContainer} />
```

Comment:
At the end of the `<Route />` on line 16 there is a space between "}" and "/>".  Update the other `Route`'s to also have this space